content/blog/2018-11-13-antarctic.md

```diff
@@ -0,0 +1,347 @@
+---
+slug: "antarctic"
+title: The Antarctic/Southern Ocean rOpenSci community
+authors:
+  - name: Ben Raymond and Michael Sumner on behalf of the community
+date: 2018-11-13
+categories: blog
+topicid: 1234
```

Comment:
```suggestion
topicid: 
twitterImg: img/blog-images/2018-11-13-antarctic/plotanim-1.gif
```

Reply:
I will add the `topicid` at time of publication.

`twitterImg` is set so when the link to this blog post is included in a tweet, that image will be associated with the tweet.